### PR TITLE
ENH: make first login process more clear in the readme.md

### PR DESCRIPTION
**Pull request by isAAAc**
## Problem

- process to first login was not clear for me and for other users 

## Solution

- i described it step by step

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by isAAAc** [comment]
!testme

File: README.md [hunk 1]
```diff
@@ -44,10 +44,17 @@ Use the YunoHost admin panel to install Friendica by entering the GitHub repo ad
 
 		https://github.com/YunoHost-Apps/friendica_ynh
 
-## User with LDAP admin rights
-**For admin rights**: When installation is complete, you will need to visit your domain page and login with the **admin account username and password** which was entered at the time of installation process. You can then create your profile and access the admin panel.
-
- **For normal YunoHost users :** Normal LDAP users can login through Ldap authentication and create there profiles.
+### First Login
+as explain in the [issue #43](https://github.com/YunoHost-Apps/friendica_ynh/issues/43),
```
Comment (ericgaspar) [inline README.md:48]: Do we need to refer to this issue or the steps in the readme are clear enough?

File: README.md [hunk 2]
```diff
@@ -44,10 +44,17 @@ Use the YunoHost admin panel to install Friendica by entering the GitHub repo ad
...
+as explain in the [issue #43](https://github.com/YunoHost-Apps/friendica_ynh/issues/43), 
+you need to **create a user** in the login page after the installation:
+- **same login** as the 'user admin' login choosed during the installation
+- check your mails: you'll have the **password** for this login
+- log in with **mail adress** and sended password by mail
+- you'll log in with admin rights
+- feel free to change your password
+ 
+#### For normal YunoHost users (non-admin):
+Normal LDAP users can login through Ldap authentication and create there profiles.
```
Comment (ericgaspar) [inline README.md:57]:
```suggestion
Normal LDAP users can login through LDAP authentication and create there profiles.
```

File: README.md [hunk 3]
```diff
@@ -44,10 +44,17 @@ Use the YunoHost admin panel to install Friendica by entering the GitHub repo ad
...
-## User with LDAP admin rights
-**For admin rights**: When installation is complete, you will need to visit your domain page and login with the **admin account username and password** which was entered at the time of installation process. You can then create your profile and access the admin panel.
-
- **For normal YunoHost users :** Normal LDAP users can login through Ldap authentication and create there profiles.
+### First Login
+as explain in the [issue #43](https://github.com/YunoHost-Apps/friendica_ynh/issues/43), 
+you need to **create a user** in the login page after the installation:
+- **same login** as the 'user admin' login choosed during the installation
+- check your mails: you'll have the **password** for this login
+- log in with **mail adress** and sended password by mail
```
Comment (ericgaspar) [inline README.md:52]:
```suggestion
- log in with **mail adress** and passport send by mail
```

File: README.md (hunk 1 above)
Comment (isAAAc) [reply README.md:48]: why not ?

but if not, no trouble on my side, it was just for traceability

**Comment by isAAAc** [comment]
ok with you @ericgaspar  ;)

File: README.md (hunk 1 above)
Comment (ericgaspar) [reply README.md:48]: @isAAAc I have no problem with mentioning the issue and I can revert the change if you think it's better to understand the context. My point was that your explanation summarize the issue clearly enough for not having to go thought the issue itself.

**Pull request merged**